comps/guardrails/factuality/predictionguard/requirements.txt

```diff
@@ -0,0 +1,11 @@
+aiohttp
+docarray[full]
```

Comment:
This causes issues when trying to install requirements locally on macOS. I think `docarray` without the `[full]` is sufficient.